src/main/java/citation.properties

```diff
@@ -0,0 +1,473 @@
+datasetfieldtype.timePeriodCoveredEnd.watermark=YYYY-MM-DD
+datasetfieldtype.dateOfCollection.watermark=
+datasetfieldtype.dateOfCollectionStart.watermark=YYYY-MM-DD
+datasetfieldtype.dateOfCollectionEnd.watermark=YYYY-MM-DD
+datasetfieldtype.kindOfData.watermark=
+datasetfieldtype.series.watermark=
+datasetfieldtype.seriesName.watermark=
+datasetfieldtype.seriesInformation.watermark=
+datasetfieldtype.software.watermark=
+datasetfieldtype.softwareName.watermark=
+datasetfieldtype.softwareVersion.watermark=
+datasetfieldtype.relatedMaterial.watermark=
+datasetfieldtype.relatedDatasets.watermark=
+datasetfieldtype.otherReferences.watermark=
+datasetfieldtype.dataSources.watermark=
+datasetfieldtype.originOfSources.watermark=
+datasetfieldtype.characteristicOfSources.watermark=
+datasetfieldtype.accessToSources.watermark=
+controlledvocabulary.subject.agricultural_sciences=Agricultural Sciences
+controlledvocabulary.subject.arts_and_humanities=Arts and Humanities
+controlledvocabulary.subject.astronomy_and_astrophysics=Astronomy and Astrophysics
+controlledvocabulary.subject.business_and_management=Business and Management
+controlledvocabulary.subject.chemistry=Chemistry
+controlledvocabulary.subject.computer_and_information_science=Computer and Information Science
+controlledvocabulary.subject.earth_and_environmental_sciences=Earth and Environmental Sciences
+controlledvocabulary.subject.engineering=Engineering
+controlledvocabulary.subject.law=Law
+controlledvocabulary.subject.mathematical_sciences=Mathematical Sciences
+controlledvocabulary.subject.medicine,_health_and_life_sciences=Medicine, Health and Life Sciences
+controlledvocabulary.subject.physics=Physics
+controlledvocabulary.subject.social_sciences=Social Sciences
+controlledvocabulary.subject.other=Other
+controlledvocabulary.publicationIDType.ark=ark
+controlledvocabulary.publicationIDType.arxiv=arXiv
+controlledvocabulary.publicationIDType.bibcode=bibcode
+controlledvocabulary.publicationIDType.doi=doi
+controlledvocabulary.publicationIDType.ean13=ean13
+controlledvocabulary.publicationIDType.eissn=eissn
+controlledvocabulary.publicationIDType.handle=handle
+controlledvocabulary.publicationIDType.isbn=isbn
+controlledvocabulary.publicationIDType.issn=issn
+controlledvocabulary.publicationIDType.istc=istc
+controlledvocabulary.publicationIDType.lissn=lissn
+controlledvocabulary.publicationIDType.lsid=lsid
+controlledvocabulary.publicationIDType.pmid=pmid
+controlledvocabulary.publicationIDType.purl=purl
+controlledvocabulary.publicationIDType.upc=upc
+controlledvocabulary.publicationIDType.url=url
+controlledvocabulary.publicationIDType.urn=urn
+controlledvocabulary.contributorType.data_collector=Data Collector
+controlledvocabulary.contributorType.data_curator=Data Curator
+controlledvocabulary.contributorType.data_manager=Data Manager
+controlledvocabulary.contributorType.editor=Editor
+controlledvocabulary.contributorType.funder=Funder
```

Comment:
"Funder" would be handy over at https://github.com/IQSS/dataverse/commit/043080dd13605277cf058097fd8e2795a00f1a4b#r30869150